src/probabilities_estimators/utils.jl

```diff
@@ -0,0 +1,27 @@
+export alphabet_length
+
+"""
+    alphabet_length(estimator) → Union{Int, Nothing}
+
+Returns the total number of possible symbols/states implied by `estimator`.
+If the total number of states cannot be known a priori, `nothing` is returned.
+Primarily used for normalization of entropy values computed using symbolic estimators.
+
+## Examples
+
+```jldoctest setup = :(using Entropies)
+julia> est = SymbolicPermutation(m = 4)
+SymbolicPermutation{typeof(Entropies.isless_rand)}(1, 4, Entropies.isless_rand)
+
+julia> alphabet_length(est)
+24
+```
+"""
+function alphabet_length end
+
+alphabet_length(est) =
+    throw(error("alphabet_length not implemented for estimator of type $(typeof(est))"))
+alphabet_length(est::SymbolicPermutation)::Int = factorial(est.m)
+alphabet_length(est::SymbolicWeightedPermutation)::Int = factorial(est.m)
+alphabet_length(est::SymbolicAmplitudeAwarePermutation)::Int = factorial(est.m)
```

Comment:
This function shoould be extended in the same file the `probabilities` function is extended, i.e., in the file of the estimator.